src/SearchImpl/Indexing/CentralizedIndexingActivityQueue.cs

```diff
@@ -0,0 +1,291 @@
+﻿using SenseNet.ContentRepository.Search;
+using SenseNet.ContentRepository.Storage;
+using SenseNet.ContentRepository.Storage.Data;
+using SenseNet.Diagnostics;
+using SenseNet.Search.Indexing.Activities;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SenseNet.Search.Indexing
+{
+    internal static class CentralizedIndexingActivityQueue
+    {
+        private static readonly int MaxCount = 10;
+        private static readonly int RunningTimeoutInSeconds = Configuration.Indexing.LuceneActivityTimeoutInSeconds;
```

Comment:
you should rename Configuration.Indexing.LuceneActivityTimeoutInSeconds to Configuration.Indexing.ActivityTimeoutInSeconds, so it will not contain the word Lucene that is nothing to do with centralized indexing

Reply:
LuceneActivityTimeoutInSeconds renamed to IndexingActivityTimeoutInSeconds